src/integrationTest/resources/application.properties

```diff
@@ -31,3 +31,6 @@ service-config.services[0].update-url=http://localhost:${wiremock.server.port}/u
 service-config.services[0].transformation-url=http://localhost:${wiremock.server.port}/transform-exception-record
 
 flyway.skip-migrations=false
+
+spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver
+spring.datasource.url=jdbc:tc:postgresql:11.0://localhost/blob_router
```

Comment:
and you need to add this to functional test application yaml. it should solve the problem.